steering_controllers_library/src/steering_controllers_library.yaml

```
@@ -23,9 +16,9 @@ steering_controllers_library:
     }
   }
 
-  front_wheels_names: {
+  steers_names: {
```

```suggestion
  steering_wheels_names: {
```
or `steering_joints_names`